packages/components/releaseNotes/components.md

```diff
@@ -1,6 +1,15 @@
 # @labkey/components
 Components, models, actions, and utility functions for LabKey applications and pages
 
+### version TBD
+*Released*: TBD
+- Parent type selector updates for adding and removing from EditableGrid
+  - AddEntityButton asButton property to allow for icon only version
+  - DropdownAnchor asAnchor property to allow for version without `<a>` tag and href
```

Comment:
```suggestion
  - DropdownAnchor 
      - Rename to DropdownMenu
      - Add asAnchor property to allow for version without `<a>` tag and href
```

Reply:
Done. Good catch. I forgot to update the release notes after I did the rename (which was part of CR feedback from Susan).